Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -51,16 +52,19 @@ final class SettingsAccountViewController: UIViewController, MessageBannerViewCo
   override func bindViewModel() {
     self.viewModel.outputs.reloadData
       .observeForUI()
-      .observeValues { [weak self] currency, shouldHideEmailWarning in
+      .observeValues { [weak self] currency, shouldHideEmailWarning, shouldHideEmailPasswordSection in
         self?.dataSource.configureRows(currency: currency,
-                                       shouldHideEmailWarning: shouldHideEmailWarning)
+                                       shouldHideEmailWarning: shouldHideEmailWarning,
+                                       shouldHideEmailPasswordSection: shouldHideEmailPasswordSection)
         self?.tableView.reloadData()
     }
 
     self.viewModel.outputs.fetchAccountFieldsError
       .observeForUI()
       .observeValues { [weak self] in
-        self?.dataSource.configureRows(currency: nil, shouldHideEmailWarning: true)
+        self?.dataSource.configureRows(currency: nil,
+                                       shouldHideEmailWarning: true,
+                                       shouldHideEmailPasswordSection: true)
```

Comment:
I would say that `shouldHideEmailPasswordSection` should default to `false` under error scenarios. We still want most users to be able to get to Change email and Change password if the settings account fields fail to load; facebook users would get an error at the change email/change password screen anyway.